source/speech/speech.py

```diff
@@ -82,13 +82,15 @@ class SpeechMode(IntEnum):
 	off = 0
 	beeps = 1
 	talk = 2
+	onDemand = 3
 
 
 @dataclass
 class SpeechState:
 	beenCanceled = True
 	isPaused = False
-	#: How speech should be handled; one of SpeechMode.off, SpeechMode.beeps or SpeechMode.talk.
+	#: How speech should be handled; one of SpeechMode.off, SpeechMode.beeps, SpeechMode.talk. or

```

Comment:
I think it is obvious that this should be one of the enum values:
```suggestion
	#: How speech should be handled.
```